examples/wagtail/core/jinja/home_page.jinja

```diff
@@ -1,6 +1,7 @@
 {% from 'govie/header/macro.html' import govieHeader %}
 {% from 'govie/heading/macro.html' import govieHeading %}
 {% from 'govie/link/macro.html' import govieLink %}
+{% from 'govie/paragraph/macro.html' import govieParagraph %}
 
```

Comment:
just FYI, the path of the imports will change once the footer branch is merged. this is as a solution to the macro imports

Reply:
Sure 👍 

Lets see who will resolve the conflicts first 😄